scripts/functions/generate_data.py

```diff
@@ -547,6 +552,12 @@ def add_experiments(
     # Create an array of the simulated data indices
     simulated_ind = np.array(simulated_data.index)
 
+    # Create list to store multiple compendia
```

Comment:
why are these commented out? if not used, perhaps delete?

Reply:
Good catch.  I have deleted these obsolete lines